core/domain/exp_services.py

```diff
@@ -943,14 +1198,32 @@ def save_exploration_summary(exp_summary):
 
 
 def delete_exploration_summary(exploration_id):
-    """Delete an exploration summary model."""
+    """Delete an exploration summary model.
+
+    Args:
+        exploration_id: str. The id of the exploration summary to be
+            deleted.
+    """
 
     exp_models.ExpSummaryModel.get(exploration_id).delete()
 
 
 def revert_exploration(
         committer_id, exploration_id, current_version, revert_to_version):
-    """Reverts an exploration to the given version number. Commits changes."""
+    """Reverts an exploration to the given version number. Commits changes.
+
+    Args:
+        committer_id: str. ID of the user who made the commit.
+        exploration_id: str. ID of the exploration to be reverted to the
+            curent version.
+        current_version: float. The present version.
```

Comment:
Version's data type is int or float? As i can see  inside "core.storage.exploration.gae_model.py" everywhere version is assigned as ndb.IntegerProperty().  @seanlip